### PR TITLE
Update

### LINKED ISSUE #60: Problem with local build of this env
The env builds okay, but I do not seem to be able to open a working browser window with

``jupyter notebook`` - just a white screen, the pasting of the suggested links into address bar does not help. The cmd is reporting intermittently non-existing kernel (note: I had to rename the to-be-created env to some other name in ``environment.yml`` as I was getting an error from mamba that ``prefix already exists``.

### PR DESCRIPTION
**Pull request by jburel**
This PR
* removes the badge for binder
* updates the notebooks to run in colab
* reviews the installation instructions
* updates the dependencies and fixes https://github.com/ome/omero-guide-python/issues/60
* replaces stardist by cellpose due to dependencies conflict. In order to support stardist a dedicated environment will have to be created

### CODE REVIEW

File: scripts/idr0062_prediction.py [hunk 1]
```diff
@@ -112,53 +111,53 @@ def load_labels_as_masks(conn, image):
...
+                coordinates.append(coordinates[0])
+                shape = Polygon(coordinates)
+                properties = {
+                    "stroke-width": 1,
+                    "z": z,
+                }
+                shapes.append(Feature(geometry=shape, properties=properties))
+
     gc = FeatureCollection(shapes)
-    return label_slices, gc
```
Comment (pwalczysko) [inline scripts/idr0062_prediction.py:154]: ``label_slices`` not defined now - error later in the notebook

File: notebooks/idr0062_prediction_zarr_public_s3.ipynb [hunk 2]
```diff
@@ -29,210 +31,397 @@
...
+    "    fig = plt.figure(figsize=(10, 10))\n",
+    "    plt.subplot(121)\n",
+    "    plt.imshow(r)\n",
+    "    plt.title(\"Cellpose z: %s\" % z)\n",
+    "    plt.subplot(122)\n",
+    "    plt.imshow(labels[0, 0, z, :, :])\n",
+    "    plt.title(\"Original z: %s\" % z)\n",
+    "    fig.canvas.flush_events()\n",
     "\n",
-    "label_slices = numpy.array(results)"
```
Comment (pwalczysko) [inline notebooks/idr0062_prediction_zarr_public_s3.ipynb:227]: ``label_slices`` undefined now

**Comment by pwalczysko** [comment]
The notebook zarr-public-s3-segmentation-parallel does not load in Colab at all, with error

```

The file has been corrupted or is not a valid notebook file.

SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
ib@https://ssl.gstatic.com/colaboratory-static/common/78fd4640f7d67f94e8a68d0d5bbcdea9/external_polymer_binary_l10n__en_gb.js:88:606
FO@https://ssl.gstatic.com/colaboratory-static/common/78fd4640f7d67f94e8a68d0d5bbcdea9/external_polymer_binary_l10n__en_gb.js:2499:164
CO/<@https://ssl.gstatic.com/colaboratory-static/common/78fd4640f7d67f94e8a68d0d5bbcdea9/external_polymer_binary_l10n__en_gb.js:2482:513
za@https://ssl.gstatic.com/colaboratory-static/common/78fd4640f7d67f94e8a68d0d5bbcdea9/external_polymer_binary_l10n__en_gb.js:16:57
ua.prototype.next_@https://ssl.gstatic.com/colaboratory-static/common/78fd4640f7d67f94e8a68d0d5bbcdea9/external_polymer_binary_l10n__en_gb.js:14:195
kaa/this.next@https://ssl.gstatic.com/colaboratory-static/common/78fd4640f7d67f94e8a68d0d5bbcdea9/external_polymer_binary_l10n__en_gb.js:17:48
b@https://ssl.gstatic.com/colaboratory-static/common/78fd4640f7d67f94e8a68d0d5bbcdea9/external_polymer_binary_l10n__en_gb.js:17:309
```

**Review by pwalczysko — CHANGES_REQUESTED** [review]
Please address two issues:

1. ``label_slices`` undefined
2. one notebook does not open in Colab

**Comment by pwalczysko** [comment]
(sorry, closed the PR by misclick) - reopened

File: scripts/idr0062_prediction.py (hunk 1 above)
Comment (jburel) [reply scripts/idr0062_prediction.py:154]: sorry. I forgot to save. I will sort it out

**Comment by jburel** [comment]
@pwalczysko the problem with  ``label_slices`` has now been fixed
The error reported in https://github.com/ome/omero-guide-python/pull/61#issuecomment-2082873000. This is not an error, it is due to the fact that the notebook is a symlink to idr0044_zarr_segmentation_parallel.ipynb

**Comment by pwalczysko** [comment]
the erro in Colab https://github.com/ome/omero-guide-python/pull/61#issuecomment-2082873000 is persisting

**Comment by pwalczysko** [comment]
> This is not an error, it is due to the fact that the notebook is a symlink to idr0044_zarr_segmentation_parallel.ipynb

maybe I do not understand - possibly I am not supposed to select that notebook in Colab at all then ? Or will it fix itself once this is merged due to symlinking to another notebook on master branch ?

**Comment by jburel** [comment]
No, you are not supposed to select it in Colab.

**Comment by pwalczysko** [comment]
> the problem with `label_slices` has now been fixed

yes, confirming, ta

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**